src/components/pages/UserPage.js

```diff
@@ -5,17 +5,20 @@ import API from '../../utils/api';
 import PropTypes from 'prop-types';
 import { useRecoilState } from 'recoil';
 import { userRepoState } from '../../recoil/atoms/userRepoState';
+import { userCommitHistoryState } from '../../recoil/atoms/userCommitHistoryState';
 
 export const UserPage = ({ match }) => {
   const { user } = match.params;
   const [userRepos, setUserRepos] = useRecoilState(userRepoState);
+  const [, setUserCommitHistory] = useRecoilState(userCommitHistoryState);
```

Comment:
Does this work for ignoring the first value? (haven't seen it but that's cool)

Reply:
yup, its strange but works. it a little better than setting an unused variable and having an eslint ignore comment